sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/BridgeInternal.java

```diff
@@ -55,257 +51,220 @@
  * This is meant to be used only internally as a bridge access to classes in
  * com.azure.cosmos
  **/
+@Warning(value = "Internal use only, not meant for public usage")
```

Comment:
maybe we should mention that these may change in future.

Reply:
Done.